src/components/Loader/InfoPerson.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Person } from '../../types/Person';
+
+type Propds = {
```

Comment:
There is a typo on line 8. 'Propds' should be replaced with 'Props'. This will make the code more readable and understandable.